### ISSUE TITLE
find out why `@strudel.cycles/eval` fails with skypack

### ISSUE THREAD

**Issue by felixroos** [tidalcycles/strudel#79]
Opening a html file with the following contents:

```html
<script type="module">
  import { evaluate } from 'https://cdn.skypack.dev/@strudel.cycles/eval';
</script>
```

throws this error:

```js
shift-parser.js:4857 Uncaught TypeError: Super expression must either be null or a function, not undefined
    at _inherits (shift-parser.js:4857:13)
    at shift-parser.js:4931:5
    at shift-parser.js:5690:4
    at createCommonjsModule (shift-parser.js:15:6)
    at shift-parser.js:4785:19
```

This seems to be an issue with `shift-parser`. The error can also be produced by:

```html
<script type="module">
  import { parseScriptWithLocation } from 'https://cdn.skypack.dev/shift-parser';
</script>
```

**Comment by felixroos** [comment]
see https://github.com/shapesecurity/shift-parser-js/issues/452

**Comment by felixroos** [comment]
wont fix, as #249 works with skypack
